config/routes.rb

```diff
@@ -3,4 +3,20 @@
 
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same comment in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**

Reply:
Alright, Thanks for the review @tresorsawasawa